CODE-REVIEW.md

```diff
@@ -53,6 +53,29 @@ ex: `DELETE /resources/{resourceId}`
 ex: `PATCH /resources/{resourceId}`
 - Em dúvidas? Mantenha uma consistência com as URLs já existem.
 
+### Padrão para códigos e mensagens de erro
+- Use mensagens bastante descritivas. Pode usar várias frases sempre que necessário!
+- Para ids inexistentes **INFORMADOS NA URL**:
+```json
+{"code": "...NotFound", "message": "The specified ... does not exist.", "status_code": 404}
+```
+- Para ids inexistentes **INFORMADOS NO REQUEST BODY**:
+```json
+{"code": "Invalid...", "message": "source ... does not exist", "status_code": 400}
+```
+- Para campos obrigatórios faltantes (não validados pelo pydantic) **INFORMADOS NO REQUEST BODY**:
+```json
+{"code": "MissingRequired...", "message": "Necessary at least ...", "status_code": 400}
+```
+- Para campos que violam restruções de valor único **INFORMADOS NO REQUEST BODY**:
+```json
+{"code": "...Exists", "message": "a ... with that ... already exists", "status_code": 400}
+```
+- Para erros internos (ex: erro em operações do Kubernetes/Kubeflow):
+```json
+{"code": "Cannot...", "message": "...", "status_code": 500}
+```
+
```

Comment:
O que vocês acham de padronizar o `"message": "..."` também? Por exemplo: 
Tem que começar com letra maiúscula e terminar com ponto final. Algo assim:
`{"code": "InvalidTaskId", "message": "source task does not exist", "status": 400}` **(ERRADO)**
`{"code": "InvalidTaskId", "message": "The specified task id does not exist.", "status": 400}` **(CERTO)**

Reply:
Acho interessante no quesito estético e de boas práticas

Reply:
Okay! :)
Vou tentar fazer os ajustes também!